clib/dpd.js

```diff
@@ -109,7 +164,7 @@
       var query = encodeIfComplex(options.query);
 
       return _dpd.ajax(root + joinPath(BASE_URL, options.path), {
-          method: "GET"
+        method: "GET"
```

Comment:
There are more spacing changes here than necessary (and in other declarations underneath). This should line up with the declaration underneath, since it's a consistent style in other deployd source files.